microprofile/opentracing-smallrye/src/main/java/org/wildfly/microprofile/opentracing/smallrye/TracingCDIExtension.java

```diff
@@ -34,8 +34,8 @@
 public class TracingCDIExtension implements Extension {
 
     public void observeBeforeBeanDiscovery(@Observes BeforeBeanDiscovery bbd, BeanManager manager) {
-        bbd.addAnnotatedType(manager.createAnnotatedType(TracerProducer.class));
-        bbd.addAnnotatedType(manager.createAnnotatedType(SmallRyeTracingCDIInterceptor.class));
+        bbd.addAnnotatedType(manager.createAnnotatedType(TracerProducer.class), TracerProducer.class.toString());
+        bbd.addAnnotatedType(manager.createAnnotatedType(SmallRyeTracingCDIInterceptor.class), SmallRyeTracingCDIInterceptor.class.toString());
```

Comment:
Is simply using the class name best practice?

In the only other example of this I see, at #10899, @stuartwdouglas incorporated the extension name in the id.

I don't see what difference it would make in this case though.

Reply:
Won't make difference here - we are not registering the same AT multiple times, if we were then the extension name would work as the identifying part.
But for the sake of safety, I can add the extension as well. Won't cause harm for sure.